src/gt4py/next/program_processors/runners/dace_iterator/__init__.py

```diff
@@ -184,43 +184,89 @@ def get_cache_id(
     return m.hexdigest()
 
 
-def run_dace_iterator(program: itir.FencilDefinition, *args, **kwargs) -> None:
+def build_sdfg_from_itir(
+    program: itir.FencilDefinition,
+    *args,
+    offset_provider: dict[str, Any],
+    auto_optimize: bool = False,
+    for_gpu: Optional[bool] = None,
+    column_axis: Optional[Dimension] = None,
+    lift_mode: LiftMode = LiftMode.FORCE_INLINE,
+    **kwargs,
+) -> dace.SDFG:
+    """Translate a Fencil into an SDFG.
+
+    Args:
+        program:	        The Fencil that should be translated.
+        *args:		        Arguments for which the fencil should be called.
+        offset_provider:	The set of offset providers that should be used.
+        auto_optimize:	    Apply DaCe's `auto_optimize` heuristic.
+        for_gpu:		    Performs the translation for GPU, defaults to `None`.
+        column_axis:		The column axis to be used, defaults to `None`.
+        lift_mode:		    Which lift mode should be used, defaults `FORCE_INLINE`.
+        **kwargs:           Except `run_on_gpu` all are ignored.
+
+    Notes:
+        In the `run_dace_iterator()` the `for_gpu` argument is called `run_on_gpu`.
+            If `for_gpu` is `None`, the default, the function will consult `run_on_gpu`.
+        Currently only the `FORCE_INLINE` liftmode is supported and an error is generated.
+    """
+    if for_gpu is None and "run_on_gpu" in kwargs:
+        for_gpu = kwargs.pop("run_on_gpu")
+    elif "run_on_gpu" in kwargs and for_gpu != kwargs["run_on_gpu"]:
+        raise ValueError(
+            "Passed conflicting arguments, for `for_gpu` you passed"
+            f"`{for_gpu}` but `run_on_gpu` was `{kwargs['run_on_gpu']}`."
+        )
```

Comment:
If the problem is that we don't like "run_" in the name of the parameter, we should rename `run_on_gpu` to `for_gpu` in the original parameter list. It's a change local to this file.